notification-portlet-webapp/src/main/java/org/jasig/portlet/notice/service/jpa/JpaNotificationDao.java

```diff
@@ -84,6 +84,22 @@ public JpaEntry getFullEntry(long entryId) {
         return query.getResultList();
     }
 
+    @Override
+    @Transactional(readOnly = true)
+    public List<JpaEntry> getNotificationsBySourceAndCustomAttribute(String source, String attributeName, String attributeValue) {
+        final String jpql = "SELECT e FROM JpaEntry e "
```

Comment:
I've reviewed a handful of JPQL references and examples.  I don't see the `AS` keyword in simple queries like these, by and large.